lib/svtplay_dl/service/urplay.py

```diff
@@ -39,8 +39,8 @@ def get(self):
                 streams = hlsparse(self.config, self.http.request("get", url), url, output=self.output)
                 for n in list(streams.keys()):
                     yield streams[n]
-                if not self.config.get("get_all_subtitles"):
-                    yield subtitle(copy.copy(self.config), "tt", stream["tt"]["location"], output=self.output)
+            if not (self.config.get("get_all_subtitles")) and (stream["default"]):
```

Comment:
no need for those extra `(` and `)`

Reply:
but i will merge it and deal with it .)

Reply:
Thanks for the feedback. Apart from a recent commit to youtube-dl, this is my first try at Python.